modules/clarity-integration/src/main/java/io/uhndata/cards/clarity/importer/ClarityImportTask.java

```diff
@@ -489,6 +515,18 @@ private Resource getOrCreateSubject(final String identifier, final String subjec
                 ClarityImportTask.PRIMARY_TYPE_PROP, "cards:Subject",
                 "identifier", identifier,
                 "type", patientType.adaptTo(Node.class)));
+            resolver.commit();
+
+            // Adjust the incrementMetricOnCreation referenced metric
+            if (!"".equals(incrementMetricOnCreation)) {
+                if (this.metricsAdjustments.get().keySet().contains(incrementMetricOnCreation)) {
+                    long oldValue = this.metricsAdjustments.get().get(incrementMetricOnCreation);
+                    this.metricsAdjustments.get().put(incrementMetricOnCreation, oldValue + 1L);
+                } else {
+                    this.metricsAdjustments.get().put(incrementMetricOnCreation, 1L);
+                }
```

Comment:
I think this would work:

```suggestion
                this.metricsAdjustments.get().compute(incrementMetricOnCreation, (k, v) -> v == null ? 1L : v + 1);
```